articles/service-fabric/service-fabric-service-manifest-resources.md

```diff
@@ -230,4 +230,4 @@ This article explained how to define endpoints in Service Fabric's service manif
 For a walk-through of packaging and deploying an existing application on a Service Fabric cluster, see:
 
 > [!div class="nextstepaction"]
-> [Package and deploy an existing executable to Service Fabric](service-fabric-deploy-existing-app.md)
\ No newline at end of file
+> [Package and deploy an existing executable to Service Fabric](service-fabric-deploy-existing-app.md)

```

Comment:
Hmm, I did not edit this, but I am seeing some sort of whitespace change?

Reply:
@listonjesse The PR review team sees this in most pull requests. I *think* you'll find information about what's happening here if you search online for 'github end of file diff'. My understanding is that it's inconsequential from the perspective of merging pull requests.